themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,83 @@
+title = "Education"
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, you can find answers to some of the most common questions on this page.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<div class="question">
+  <h3>Can I teach Godot in my classroom?</h3>
+  <p>Godot&nbsp;Engine is <a href="https://en.wikipedia.org/wiki/Free_and_open_source_software">free and open source
+      software</a> released under the permissive <a href="/license">MIT license</a>.<br>
+    You can use Godot in your classroom without any restrictions and with zero licensing
+    costs.
+  </p>
+  <p>You can read more about <a href="/license">our license here</a>.</p>
+</div>
+
+<div class="question">
+  <h3>Will the Godot project sign a services agreement provided by my institution?</h3>
+  <p>The Godot Engine project is unable to accept any terms and conditions outside of the MIT license, nor will it
+    provide warranties or representations about the Godot Engine outside those of the <a
+      href="/license">MIT license</a>.</p>
+  <p>As a free and open source project, Godot is not backed by a corporation or any individual. Accordingly, it doesn't
+    have the legal capacity to enter into agreements, even if such agreements did not conflict with the MIT license.</p>
+</div>
+
+<div class="question">
+  <h3>What data does the Godot Engine collect from users?</h3>
+  <p>When you use the Asset Library in the Godot Engine application, HTTPS requests are made to the Asset Library
+    website, and will contain: Godot Engine version, search query, selected assets, and the user's IP address. The Asset
+    Library
+    processes this data without storing it, but our TuxFamily hosting's HTTP server may log these requests. Such logs
+    are subject to TuxFamily's privacy policy (<a href="https://faq.tuxfamily.org/Privacy/En">https://faq.tuxfamily.org/Privacy/En</a>).<br>
```

Comment:
Why not just 

```suggestion
    are subject to <a href="https://faq.tuxfamily.org/Privacy/En">TuxFamily's privacy policy</a>.<br>
```

Reply:
I left the whole thing because it was an outgoing link, but I'm fine with that edit as well 👍

Reply:
`target="_blank"` then? 🐱‍👤 I'm not sure we have styling for outgoing links on the website, but we can probably add it at some point, like we have in the docs.